bin/hdfs-mesos-namenode

```diff
@@ -18,23 +18,17 @@ fi
 trap "{ $DIR/mesos-killtree "$$" ; exit 0; }" EXIT
 
 function bootstrap_standby() {
-  echo "Pausing for 10s before bootstrapping standby namenode"
```

Comment:
I'm fine with getting rid of the sleep commands, but this needs to be consistent for all scripts so the other scripts also need to be modified.